src/overlays/actors/ovl_Boss_06/z_boss_06.c

```diff
@@ -579,10 +578,11 @@ void Boss06_Draw(Actor* thisx, GlobalContext* globalCtx2) {
         Matrix_InsertTranslation(0.0f, 0.0f, -1112.0f, MTXMODE_APPLY);
 
         gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-        gDPSetPrimColor(POLY_XLU_DISP++, 0, 0x9B, 255, 255, (u8)((140.0f * sp68) + 115.0f), spD3);
+        gDPSetPrimColor(POLY_XLU_DISP++, 0, 0x9B, 255, maxColor, (u8)((140.0f * sp68) + 115.0f), spD3);
         gSPDisplayList(POLY_XLU_DISP++, object_knight_DL_018CF0);
-        gDPSetPrimColor(POLY_XLU_DISP++, 0, 0xFF, 255, 255, (u8)((100.0f * sp68) + 65.0f), spD2);
+        gDPSetPrimColor(POLY_XLU_DISP++, 0, 0xFF, 255, maxColor, (u8)((100.0f * sp68) + 65.0f), spD2);
         gSPDisplayList(POLY_XLU_DISP++, object_knight_DL_018DE0);
```

Comment:
I suspect we don't do this currently, but this looks like the `l` argument of `gDPSetPrimColor` should be dec too.